cmd/bootstrap/run/qc.go

```diff
@@ -94,48 +122,29 @@ func createValidators(participantData *ParticipantData, identities flow.Identity
 
 	groupSize := uint(len(participantData.Participants))
 	if groupSize < uint(n) {
```

Comment:
I think this check is legacy code from when we used to handle DKG participants and staking identities completely separate. By now, we compare the same quantity: 
* https://github.com/onflow/flow-go/blob/dd0239af52df8c4d559266e4b9c389592aef36a0/cmd/bootstrap/run/qc.go#L115
* https://github.com/onflow/flow-go/blob/dd0239af52df8c4d559266e4b9c389592aef36a0/cmd/bootstrap/run/qc.go#L123

### suggestion
I would suggest to combine `createSigners` with `GenerateRootBlockVotes`.  This will consolidate the logic and make it apparent that the check here is unnecessary.
**This is addresses in my PR**